app/workers/tasks.php

```diff
@@ -17,6 +17,10 @@
 
 class TasksV1 extends Worker
 {
+    public function getWorkerName(): string {
```

Comment:
```suggestion
    public function getName(): string {
```

The word `worker` is already obvious from the context. Not need to mention it again.

Reply:
Renamed all `getWorkerName` to `getName` - in abstract class, in every worker and in log where I used this method.